examples/install-firmware/doc.go

```diff
@@ -2,23 +2,57 @@
 install-firmware is an example commmand that utilizes the 'v1' bmclib interface
 methods to flash a firmware image to a BMC.
 
-	$ go run ./examples/v1/install-firmware/main.go -h
-	Usage of /tmp/go-build2950657412/b001/exe/main:
-		-cert-pool string
-					Path to an file containing x509 CAs. An empty string uses the system CAs. Only takes effect when --secure-tls=true
-		-firmware string
-					The local path of the firmware to install
-		-host string
-					BMC hostname to connect to
-		-password string
-					Username to login with
-		-port int
-					BMC port to connect to (default 443)
-		-secure-tls
-					Enable secure TLS
-		-user string
-					Username to login with
-		-version string
-					The firmware version being installed
+Note: The example installs the firmware and polls until the status until the install is complete,
+and if required by the install process - power cycles the host.
+
+							$ go run ./examples/v1/install-firmware/main.go -h
+							Usage of /tmp/go-build2950657412/b001/exe/main:
+								-cert-pool string
```

Comment:
It looks like all these flags have a single dash, but should have two?

Reply:
The flags package accepts both single and double dashes, I updated the examples to be consistent.